cmd/ketch/builder_set.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/shipa-corp/ketch/cmd/ketch/configuration"
+)
+
+const builderSetHelp = `
+Manage the default builder to be used when deploying from source. This value can also be set manually in the config.toml
+by specifying "default-builder".`
+
+func newBuilderSetCmd(ketchConfig configuration.KetchConfig) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set",
+		Short: "set default builder",
+		Long:  builderSetHelp,
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return setDefaultBuilder(ketchConfig, args[0])
+		},
+	}
+	return cmd
+}
+
+func setDefaultBuilder(ketchConfig configuration.KetchConfig, defaultBuilder string) error {
+	ketchConfig.DefaultBuilder = defaultBuilder
+	path, err := configuration.DefaultConfigPath()
+	if err != nil {
+		return err
+	}
+	return configuration.Write(ketchConfig, path)
```

Comment:
This will overwrite any other data currently in the ketch-config file, e.g. additional-builders. Should we do a get & update thing here?

Reply:
It does overwrite in the sense that the whole config.toml is rewriiten, but the function takes in the current configuration and modifies just the default-builder before printing the config back out to the file.  This makes it so that there should be no loss of the user defined additional-builders when setting the default-builder.

Reply:
I completely missed line 27. Ship it!